<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры моделирования</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Время моделирования:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb_time">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Время обслуживания в автомате: </string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_auto">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_auto_2">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>0.900000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_window">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_window_2">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Время обслуживания  в кассе: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_client">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.300000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dspb_time_client_2">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>80</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Время прихода пассажиров: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_model">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Моделировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты моделирования</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Касса</string>
     </property>
     <widget class="QLabel" name="lbl_done_window">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_queue_window">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Автомат 1</string>
     </property>
     <widget class="QLabel" name="lbl_queue_auto1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_done_auto1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_to_window_auto1">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Перешли в кассу:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Автомат 2</string>
     </property>
     <widget class="QLabel" name="lbl_to_window_auto2">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Перешли в кассу:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_queue_auto2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_done_auto2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Автомат 3</string>
     </property>
     <widget class="QLabel" name="lbl_to_window_auto3">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>21</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Перешли в кассу:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_queue_auto3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_done_auto3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Автомат 4</string>
     </property>
     <widget class="QLabel" name="lbl_to_window_auto4">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>19</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Перешли в кассу:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_queue_auto4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_done_auto4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Всего</string>
     </property>
     <widget class="QLabel" name="lbl_to_window_all">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Проход сразу к кассе:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_queue_all">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>В очереди: </string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_done_all">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обслужено:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
